pkg/config/config_test.go

```diff
@@ -620,7 +621,7 @@ image_copy_tmp_dir="storage"`
 			oldContainerConnection, hostEnvSet := os.LookupEnv("CONTAINER_CONNECTION")
 			os.Setenv("CONTAINER_CONNECTION", "QB")
 
-			u, i, err := cfg.ActiveDestination()
+			u, i, _, err := cfg.ActiveDestination()
```

Comment:
Testing that it is `false` in the various cases here would protect us against unintended accidents.